context/brand-context/default/__tests__/unit/util/onetrust.spec.js

```diff
@@ -0,0 +1,46 @@
+import {checkConsent, isConsentBannerClosed} from "../../../js/helpers/util/onetrust";
```

Comment:
How many teams are still using OneTrust? Is this still useful?

Reply:
I did wonder that, but I wanted the first commit to just be a direct copy of what we have already, if I start altering stuff then deprecate the old components it might get confusing as stuff disappears/changes